spec.emu

```diff
@@ -136,189 +179,307 @@ location: https://github.com/tc39/proposal-decimal/
-      <emu-production name="DigitString" type="lexical">
-        <emu-rhs><emu-prodref href="#prod-Decimal128Digit"></emu-prodref>+</emu-rhs>
-      </emu-production>
-      <emu-production name="UnsignedDecimal128DigitString" type="lexical" oneof>
-        <emu-rhs><emu-prodref href="#prod-Decimal128Digits"></emu-prodref></emu-rhs>
-        <emu-rhs><emu-prodref href="#prod-Decimal128Digits"></emu-prodref> "." <emu-prodref href="#prod-Decimal128Digits"></emu-prodref></emu-rhs>
-      </emu-production>
-      <emu-production name="SignedDecimal128DigitString" type="lexical" oneof>
-        <emu-rhs>-? <emu-prodref name="UnsignedDecimal128DigitString"></emu-prodref></emu-rhs>
-      </emu-production>
-    </emu-grammar>
-    <emu-grammar>
-      <emu-production name="UnsignedDecimal128DigitString" type="lexical" oneof>
-        <emu-rhs><emu-prodref href="#prod-Decimal128Digits"></emu-prodref></emu-rhs>
-        <emu-rhs><emu-prodref href="#prod-Decimal128Digits"></emu-prodref> "." <emu-prodref href="#prod-Decimal128Digits"></emu-prodref></emu-rhs>
-      </emu-production>
-      <emu-production name="SignedDecimal128DigitString" type="lexical" oneof>
-        <emu-rhs>[+-]? <emu-prodref name="UnsignedDigitString"></emu-prodref></emu-rhs>
-      </emu-production>
-    </emu-grammar>
-    <emu-grammar name="exponential-syntax">
-      <emu-production name="ExponentialDigitStringLHS" type="lexical" oneof>
-        <emu-rhs><emu-prodref href="#prod-NonZeroDigit"></emu-prodref></emu-rhs>
-        <emu-rhs><emu-prodref href="#prod-NonZeroDigit"></emu-prodref> "." <emu-prodref href="#prod-Decimal128Digits"></emu-prodref></emu-rhs>
-      </emu-production>
-      <emu-production name="ExponentialDigitStringRHS" type="lexical">
-        <emu-rhs>[-+]? <emu-prodref href="#prod-Decimal128Digits"></emu-prodref></emu-rhs>
-      </emu-production>
-    </emu-grammar>
-    <emu-grammar name="infinity" oneof>
-      <emu-production>`-` ? `Inf` [> or any token that is an ASCII case-insensitive match]</emu-production>
-      <emu-production>`-` ? `Infinity` [> or any token that is an ASCII case-insensitive match]</emu-production>
-    </emu-grammar>
-    <emu-grammar name="nan">
-      <emu-production>("N" | "n") | ("A" | "a") | ("N" | "n")</emu-production>
-    </emu-grammar>
-    <p>If *_x_* does not match any of these syntaxes, throw a *SyntaxError*.</p>
-    <p>Multiple leading zeros will be reduced to a single zero.</p>
+    <emu-clause id="sec-the-decimal128-constructor-value">
+      <h1>Decimal128 ( _x_ [ , _options_ ] )</h1>
+      <emu-alg>
+        1. If NewTarget is *undefined*, throw a *TypeError* exception.
+        1. Let _n_ be ? ToDecimal128(_x_).
+        1. Let _O_ be ? OrdinaryCreateFromConstructor(NewTarget, *"%Decimal128.prototype%"*, « [[Decimal128Data]] »).
+        1. Set _O_.[[Decimal128Data]] to _n_.
+        1. Return _O_.
+      </emu-alg>
+      <p>This function performs the following steps when called:</p>
```

Comment:
This line usually goes before the `<emu-alg>`. The following two lines are now no longer needed.